src/ai-store/browser.js

```diff
@@ -40,9 +40,9 @@ export async function getBrowserServiceData() {
 async function getBrowserAiCapabilities( ai ) {
 	const capabilities = [];
 
-	if ( ai.canCreateTextSession ) {
-		const supportsTextGeneration = await ai.canCreateTextSession();
-		if ( supportsTextGeneration && supportsTextGeneration === 'readily' ) {
+	if ( 'ai' in self && 'assistant' in self.ai) {
+		const supportsTextGeneration = await self.ai.assistant.capabilities();
```

Comment:
`window.ai` is passed to this function and is already safe to be truthy at this point, so I think this can become:
```suggestion
	if ( ai.assistant ) {
		const supportsTextGeneration = await ai.assistant.capabilities();
```